libafl_libfuzzer/runtime/src/options.rs

```diff
@@ -358,6 +363,47 @@ impl<'a> LibfuzzerOptionsBuilder<'a> {
                         }
                         "runs" => self.runs = parse_or_bail!(name, value, usize),
                         "close_fd_mask" => self.close_fd_mask = parse_or_bail!(name, value, u8),
+                        "help" => {
```

Comment:
This will go out of date at some time but I don't have a good solution either

Reply:
Yeah unfortunately using e.g. clap is blocked on https://github.com/clap-rs/clap/issues/1210 and I don't know of another good arg parsing library that supports this arg format